feature-libs/product/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@spartacus/product",
+  "version": "2.1.0-dev.0",
+  "description": "Product library for Spartacus",
+  "homepage": "https://github.com/SAP/spartacus",
+  "keywords": [
+    "spartacus",
+    "framework",
+    "storefront",
+    "product"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/SAP/spartacus",
```

Comment:
Should we point to the specific directory, like we do in core libs?
Missed that for my-account as well